client/src/main/scala/caliban/client/CalibanClientError.scala

```diff
@@ -28,20 +29,9 @@ object CalibanClientError {
    */
   case class ServerError(errors: List[GraphQLResponseError]) extends CalibanClientError {
 
-    override def toString: String =
-      s"Server Error: ${errors
-        .map(e =>
-          s"${e.message} ${e.locations
-            .getOrElse(Nil)
-            .map(loc => s"at line ${loc.line} and column ${loc.column}")
-            .mkString(" ")}${e.path.fold("")(p =>
-            s" at path ${p.map {
-              case Left(value)  => s"/$value"
-              case Right(value) => s"[$value]"
-            }.mkString("")}"
-          )}"
-        )
-        .mkString("\n")}"
-  }
+    override def toString: String = s"Server Error: ${errors.map(_.print).mkString("\n")}"
+
+    override def getFullMessage = s"Server Error: ${errors.map(_.fullPrint).mkString("\n")}"
```

Comment:
Same as below, change this to `render` and add the `includeExtensions` argument. Also change the definition of `toString` to:

`override def toString: String = render(false)`

Reply:
Sure. 
Please have another look.